### ISSUE TITLE
Typo in class name

### ISSUE THREAD

**Issue by adamdunkley** [chemerisuk/better-placeholder-polyfill#3]
There's a typo in the class name, `better-placehoder`… I presume it was meant to be `better-placeholder`.

Took me a while to work out as I was trying to apply a better z-index to my placeholder.

Potentially move over to `better-placeholder` and leave `better-placehoder` in for a version or two with a notice up in the readme here saying it will be removed?

**Comment by chemerisuk** [comment]
@adamdunkley you are right, there is a typo. I'll fix it asap and publish a new version.

>  to apply a better z-index to my placeholder
Why do you need to do it? By default z-index equals to value of the input plus 1

**Comment by adamdunkley** [comment]
@chemerisuk In the end I didn't actually need it (I removed the z-index override) I was just toying around with various things as it wasn't working properly. At the point of posting my original comment I hadn't found the solution … it was to do with how the underlying input element was sized (though it was very late night and I cannot remember the precise details: if I revisit it I'll write up what caused it to go wrong but it was definitely a strange edge case).

**Comment by adamdunkley** [comment]
Also, thanks for the polyfill (it's much better than all the other placeholder polyfills around)!
